rst2pdf/tests/input/test_issue_266.txt

```diff
@@ -1,34 +0,0 @@
-Image rescaling test
-===============================
-
-
-Normal size PDF image:
-
-.. image:: images/text_and_vectors_001.pdf
-
-
-Shrunken PDF image:
-
-.. image:: images/text_and_vectors_001.pdf
-   :scale: 60%
-
-Enlarged PDF image:
-
-.. image:: images/text_and_vectors_001.pdf
-   :scale: 200%
-
-
-Normal image:
-
-.. image:: images/middleware.jpg
```

Comment:
Is there value in keeping a test with rescaled JPGs? I realise the PDFs won't work

Reply:
Very good point. I've restored as `test_jpg_scaling`.